pennylane/ops/op_math/dot.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
```

Comment:
Should this be 2023 now? :)

Reply:
Yes, definitely! Out of curiosity, do we need to update all the copyright messages in every file? 😮

Reply:
Updated!

Reply:
@josh146 how do we deal with this?

Reply:
> Out of curiosity, do we need to update all the copyright messages in every file?

Nope. Only files that are updated we can consider updating the copyright, but even then I do not believe it is necessary --- copyright automatically applies until you explicitly license it otherwise.

For files that are not modified, extending/changing the date in the file does not extend the copyright.